waypoints-special/CAN_PG_A-M.cup

```diff
@@ -1,415 +1,414 @@
 ObsZone=3,Style=1,R1=2000m,A1=180
-ObsZone=4,Style=3,R1=2000m,A1=180
-"AVVC 2 - Ste Rosalie","CSS4","Ste Rosalie","St Valerien","Ste Helene","St Dominique City","Ste Rosalie","CSS4"
+ObsZone=4,Style=3,R1=2000m,A1=180,Move=1
+"AVVC 2 - Ste Rosalie","CSS4","Ste Rosalie","St Valerien","Ste Helene","St Dominique City","Ste Rosalie","CSS4",,,
 Options,WpDis=False,NearDis=0.0km,NearAlt=0.0ft
-ObsZone=0,Style=2,R1=2000m,A1=180
+ObsZone=0,Style=2,R1=2000m,A1=180,Move=1
 ObsZone=1,Style=1,R1=2000m,A1=180
 ObsZone=2,Style=1,R1=2000m,A1=180
 ObsZone=3,Style=1,R1=2000m,A1=180
-ObsZone=4,Style=3,R1=2000m,A1=180
-"AVVC 2inv - St Dominique","CSS4","St Dominique City","Ste Helene","St Valerien","Ste Rosalie","St Dominique City","CSS4"
+ObsZone=4,Style=3,R1=2000m,A1=180,Move=1
+"AVVC 2inv - St Dominique","CSS4","St Dominique City","Ste Helene","St Valerien","Ste Rosalie","St Dominique City","CSS4",,,
 Options,WpDis=False,NearDis=0.0km,NearAlt=0.0ft
-ObsZone=0,Style=2,R1=2000m,A1=180
+ObsZone=0,Style=2,R1=2000m,A1=180,Move=1
 ObsZone=1,Style=1,R1=2000m,A1=180
 ObsZone=2,Style=1,R1=2000m,A1=180
 ObsZone=3,Style=1,R1=2000m,A1=180
-ObsZone=4,Style=3,R1=2000m,A1=180
-"AVVC 2inv - St Valerien","CSS4","St Valerien","Ste Rosalie","St Dominique City","Ste Helene","St Valerien","CSS4"
+ObsZone=4,Style=3,R1=2000m,A1=180,Move=1
+"AVVC 2inv - St Valerien","CSS4","St Valerien","Ste Rosalie","St Dominique City","Ste Helene","St Valerien","CSS4",,,
 Options,WpDis=False,NearDis=0.0km,NearAlt=0.0ft
-ObsZone=0,Style=2,R1=2000m,A1=180
+ObsZone=0,Style=2,R1=2000m,A1=180,Move=1
 ObsZone=1,Style=1,R1=2000m,A1=180
 ObsZone=2,Style=1,R1=2000m,A1=180
 ObsZone=3,Style=1,R1=2000m,A1=180
-ObsZone=4,Style=3,R1=2000m,A1=180
-"AVVC 2inv - Ste Helene","CSS4","Ste Helene","St Valerien","Ste Rosalie","St Dominique City","Ste Helene","CSS4"
+ObsZone=4,Style=3,R1=2000m,A1=180,Move=1
+"AVVC 2inv - Ste Helene","CSS4","Ste Helene","St Valerien","Ste Rosalie","St Dominique City","Ste Helene","CSS4",,,
 Options,WpDis=False,NearDis=0.0km,NearAlt=0.0ft
-ObsZone=0,Style=2,R1=2000m,A1=180
+ObsZone=0,Style=2,R1=2000m,A1=180,Move=1
 ObsZone=1,Style=1,R1=2000m,A1=180
 ObsZone=2,Style=1,R1=2000m,A1=180
 ObsZone=3,Style=1,R1=2000m,A1=180
-ObsZone=4,Style=3,R1=2000m,A1=180
-"AVVC 2inv - Ste Rosalie","CSS4","Ste Rosalie","St Dominique City","Ste Helene","St Valerien","Ste Rosalie","CSS4"
+ObsZone=4,Style=3,R1=2000m,A1=180,Move=1
+"AVVC 2inv - Ste Rosalie","CSS4","Ste Rosalie","St Dominique City","Ste Helene","St Valerien","Ste Rosalie","CSS4",,,
 Options,WpDis=False,NearDis=0.0km,NearAlt=0.0ft
-ObsZone=0,Style=2,R1=2000m,A1=180
+ObsZone=0,Style=2,R1=2000m,A1=180,Move=1
 ObsZone=1,Style=1,R1=2000m,A1=180
 ObsZone=2,Style=1,R1=2000m,A1=180
 ObsZone=3,Style=1,R1=2000m,A1=180
-ObsZone=4,Style=3,R1=2000m,A1=180
-"AVVC 3","CSS4","CSS4","St Hyacinthe","Drummondville Airport","CSS4","CSS4"
+ObsZone=4,Style=3,R1=2000m,A1=180,Move=1
+"AVVC 3","CSS4","CSS4","St Hyacinthe","Drummondville Airport","CSS4","CSS4",,,,
```

Comment:
This is what the parser is falling over. The repeated waypoints. It cannot parse a course from the same location to the same location again.

Reply:
At least in my version of SeeYou, the first CSS4 is the takeoff point and the second is the start.

Reply:
And the last CSS4s are finish and landing point